python/adjoint/optimization_problem.py

```diff
@@ -130,7 +130,7 @@ def __init__(
                     2))  # index of center frequency
 
         self.decay_by = decay_by
-        self.decay_fields = decay_fields
+        self.decay_fields = decay_fields # left for legacy, but doesn't do anything
```

Comment:
Probably better to just remove this line here and also from `test_adjoint_solver.py` where it is also used?

Reply:
Fixed.